app/screens/main/Main.js

```diff
@@ -94,10 +95,17 @@ class Main extends Component<Props, State> {
   }
 
   render() {
+    const { isNetworkConnected } = this.props;
     const { selectedItemIndex, loadingItemIndex } = this.state;
     return (
       <Wrapper>
-        <SideMenu items={sideMenuItems} selectedItemIndex={selectedItemIndex} onMenuItemPress={this.handleSideMenuPress} loadingItemIndex={loadingItemIndex} />
+        <SideMenu
+          items={sideMenuItems}
+          networkDisconnectedIndex={isNetworkConnected ? -1 : 5}
```

Comment:
no need to pass parameter like this, looks ugly. Just connect SideMenu to store and pass param there

Reply:
ok